infrastructure/OffboardingWorkflow.md

```diff
@@ -1,12 +1,11 @@
 # Workflow for offboarding members from services (requires admin rights to all resources)
-
-To offboard ,a member (or someone from their team) must:
- - Fill out the [offboarding form](https://forms.gle/Gb5H6CAYCd8iVYKG8)
+To offboard, a member (or someone from their team) must:
+ - Fill out the [offboarding form](https://www.nih-cfde.org/offboarding-form/) on the CFDE website (login required).
  
- We track those events, as well as our progress in adding them to our resources [here](https://docs.google.com/spreadsheets/d/1bY-P5ZrIOPDrDpzlXNrDnV96XqOnV_X9z9ur7t9Qkhk/edit?usp=sharing).
+We track those events, as well as our progress in adding them to our resources [here](https://docs.google.com/spreadsheets/d/1bY-P5ZrIOPDrDpzlXNrDnV96XqOnV_X9z9ur7t9Qkhk/edit?usp=sharing).
 
 ## Offboarding Overview
-The Offboarding form will send notifications to coordination@CFDE.groups.io whenever the form is filled out. To request being added to coordination mailing list and receive notifications, send your request to the [helpdesk](mailto:coordination+int+1482+3914664481228082529@CFDE.groups.io).
+Offboard requests are handled by the [offboarding form](https://www.nih-cfde.org/offboarding-form/) on the [CFDE Website](https://www.nih-cfde.org/). Responses are recorded and sent to the [CFDE HelpDesk](support@cfde.atlassian.net).
 
 A member of the coordination mailing list should begin the onboarding process for new forms *within 2 business days*. Full onboarding for each person is a typically a multi-day process, and can be done by multiple memebers of the coordination team, because we have to wait for each new onboardee to accept the invitations. The basic workflow is:
```

Comment:
```suggestion
A member of the coordination mailing list should begin the onboarding process for new forms *within 2 business days*. Full onboarding for each person is typically a multi-day process, and can be done by multiple members of the coordination team, because we have to wait for each new onboardee to accept the invitations. The basic workflow is:
```